service/src/main/java/uk/gov/hmcts/reform/fpl/service/MigrateCaseService.java

```diff
@@ -417,6 +418,31 @@ public Map<String, Object> renameApplicationDocuments(CaseData caseData) {
         return Map.of("applicationDocuments", updatedList);
     }
 
+    public Map<String, Object> removeSkeletonArgument(CaseData caseData, String skeletonArgumentId,
+                                                      String migrationId) {
+        Long caseId = caseData.getId();
+        List<Element<SkeletonArgument>> skeletonArguments = caseData.getHearingDocuments().getSkeletonArgumentList();
+
+        if (skeletonArguments == null) {
```

Comment:
Should this be a null check and not skeletonArguments.isEmpty() instead?

Looks like we have `defaultIfNull(skeletonArgumentList, new ArrayList<>());` on getSkeletonArgumentList so will it not always be an arraylist?